Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml.cs

```diff
@@ -113,5 +120,33 @@ private void xBrowseImageLoadButton_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void XBusinessFlowControl_ElementChangedPageEvent()
+        {            
+            if (xBusinessFlowControl != null && xBusinessFlowControl.BusinessFlow != null)
+            {
+                mWizard.mPomLearnUtils.POM.MappedBusinessFlow = xBusinessFlowControl.BusinessFlowRepositoryKey;
+            }
+        }
+
+        private void xPageUrlRadioBtn_Checked(object sender, RoutedEventArgs e)
+        {
+            SetControlsVisibility(false);
```

Comment:
for what 2 events? you can have only single Checked event handler for both radio btns